tests/fixtures.py

```diff
@@ -84,6 +84,7 @@ def get_viv_extractor(path):
         vw = capa.main.get_workspace(path, "sc64", sigpaths=sigpaths)
     else:
         vw = capa.main.get_workspace(path, "auto", sigpaths=sigpaths)
+    vw.saveWorkspace()
```

Comment:
unrelated enhancement: save viv files during testing, which may help during local development of capa?